packages/build/src/steps/plugin.js

```diff
@@ -7,7 +7,7 @@ import { getSuccessStatus } from '../status/success.js'
 import { getPluginErrorType } from './error.js'
 import { updateNetlifyConfig, listConfigSideFiles } from './update_config.js'
 
-export const isTrustedPlugin = (pluginPackageJson) => pluginPackageJson?.name?.startsWith('@netlify/')
+export const isTrustedPlugin = (packageName) => packageName?.startsWith('@netlify/')
```

Comment:
Changed the method so that we can pass just a string and re-use across other places.